jpmobile.gemspec

```
@@ -9,9 +9,11 @@ Gem::Specification.new do |gem|
   gem.email         = ['rust.stnard@gmail.com']
   gem.description   = 'A Rails plugin for mobile devices in Japan'
   gem.summary       = 'Rails plugin for mobile devices in Japan'
-  gem.homepage      = 'http://jpmobile-rails.org'
+  gem.homepage      = 'https://jpmobile-rails.org'
   gem.license       = 'MIT'
 
+  gem.metadata['source_code_uri'] = 'https://github.com/jpmobile/jpmobile'
```

もしドキュメントのリンクがソースコードのREADMEで良ければ、こう変更しちゃいましょう。

```suggestion
  gem.metadata['source_code_uri'] = 'https://github.com/jpmobile/jpmobile'
  gem.metadata['documentation_uri'] = gem.metadata['source_code_uri']
```

---

そうですね、それでよさそうです、ありがとうございます 🙏